src/pages/GridAutoFlow.vue

```diff
@@ -0,0 +1,49 @@
+<script setup>
+// Note there's a way we can just import these globally so we don't have to do it in every page
+import Box from '../components/Box.vue'
+import Container from '../components/Container.vue'
+
+const exampleClasses = 'p-4 rounded-2 text-white flex items-center justify-center'
+</script>
+<style>
+.grid-flow-row-dense{
```

Comment:
Did this class not work for some reason?

Reply:
I assumed that it was not included in that Uno Config, but might be mistaken :) There might be stuff we dont wanna support in these Examples to, we just delete whenever when we get a better grip on the situation me thinks :D

Reply:
Makes sense!